include/libopencm3/stm32/l0/rcc.h

```diff
@@ -539,6 +549,7 @@ enum rcc_periph_clken {
 	SCC_SRAM = _REG_BIT(0x40, 12),
 	SCC_CRC = _REG_BIT(0x40, 12),
 	SCC_TOUCH = _REG_BIT(0x40, 16),
+	SCC_TSC = _REG_BIT(0x40, 16),
```

Comment:
Caught :) yes, TSC is the ref man consistent name here.  iirc TOUCH was already the name used by L1.  open for debate here.